x-pack/plugins/infra/public/pages/metrics/inventory_view/lib/color_from_value.ts

```diff
@@ -68,8 +68,8 @@ export const calculateSteppedGradientColor = (
 
   // Since the stepped legend matches a range we need to ensure anything outside
   // the max bounds get's the maximum color.
-  if (gte(normalizedValue, (last(rules) as any).value)) {
-    return (last(rules) as any).color;
+  if (gte(normalizedValue, last(rules)!.value)) {
+    return last(rules)!.color;
```

Comment:
We can help the compiler by giving it a name here:

```suggestion
  const lastRule = last(rules);
  if (lastRule && gte(normalizedValue, lastRule.value)) {
    return lastRule.color;
```